spec/forms/hyrax/forms/resource_form_spec.rb

```diff
@@ -24,6 +24,13 @@
     end
   end
 
+  describe '.model_class' do
+    it 'has a model class' do
```

Comment:
`it "is the given work's class"` might read better.

Reply:
how is that?

Reply:
As originally written this sounds like a tautology. To me, it seems that my recommendation explains, a bit better, what is happening.

Reply:
sorry. misunderstanding. :)

i changed the text, and was asking if the new version looks okay to you.